.gitignore

```diff
@@ -1,13 +1,13 @@
 node_modules
 npm-debug.log
 .DS_Store
-
 .idea
 
 build/
 example/example.js
 example/example.js.map
-package-lock.json
 demo/
 styles/css/
-yarn.lock
+
+## generated npm packages
+*-[0-9].[0-9].[0-9].tgz
```

Comment:
Not sure what this does, if it is important please open a separate PR to explain the purpose of this.

Reply:
Because we currently use cloudsmith to publish and try our changes in our apps. And `npm pack` creates `tgz` files with this name pattern: https://docs.npmjs.com/cli/pack.html. That's why. I don't see any harm in adding this, and no benefit in creating a PR with just this single line of code (if you agree of course).

Reply:
There is no harm, but again it becomes yet another thing that is added "just because" and this is very specific to your current build process. I'm trying to keep things explicit where everyone can look at it and immediately know why its there.

Take a look at some other open-source (e.g. https://github.com/reduxjs/redux) libraries and you'll see that there is very little ambiguity in "why" this line is there.

Reply:
Well, that's fair enough, I'll remove it. 👍